src/app/components/stacks-transaction-item/stacks-transaction-item.tsx

```diff
@@ -78,19 +99,40 @@ export function StacksTransactionItem({
   const txIcon = transaction ? <StacksTransactionIcon transaction={transaction} /> : icon;
   const txTitle = transaction ? getTxTitle(transaction) : title || '';
   const txValue = transaction ? getTxValue(transaction, isOriginator) : value;
-  const increaseFeeButton = (
-    <IncreaseFeeButton
+
+  const actionButtonGroup = [
```

Comment:
I think it's actually cleaner to just have 2 buttons, one for each action rather than mapping. 

Mapping is good to keep the code DRY but with just two options I'm not sure if it's worth it